frontend/src/components/InquiryList.js

```diff
@@ -0,0 +1,35 @@
+import styled from 'styled-components/macro'
+
+export default function InquiryList({inquiries}){
+
+    return(
+        <List>
+            {inquiries.map((inquiry) => (
+                <li key={inquiry.id}>
+                    <span>{inquiry.partName}</span>
+                    {/*<Link to={`/user/${user.name}`}>*/}
+                    {/*    <img src={user.avatar} alt={user.name} />*/}
+                    {/*    <span className="user-name">{user.name}</span>*/}
+                    {/*</Link>*/}
```

Comment:
- are you using this Link? if not, you may want to remove it incl. the styling down below
- if you want to re-introduce this section, it may make sense to put it into a separate component